client/src/components/Address/Address.js

```diff
@@ -41,7 +74,65 @@ const Address = () => {
         setMessage("Sorry! Something went wrong. Please Try again", err);
       });
   }, []);
+  const displayRazorPay = async () => {
+    const res = await loadRazorPay(
+      "https://checkout.razorpay.com/v1/checkout.js"
+    );
+    if (!res) {
+      alert("something went wrong ! cant load razor pay,Check internet");
+      return;
+    }
+    var options = {
+      key: process.env.RAZOR_PAY_KEY,
+      amount: response.amount.toString(),
+      currency: response.currency,
+      name: "E-commerence",
+      description: "Transaction for placing an order",
+      order_id: response.id,
+      handler: function (response) {
+        alert("successfull");
+        alert(response.razorpay_payment_id);
+        alert(response.razorpay_order_id);
+        alert(response.razorpay_signature);
+      },
+
+      prefill: {
+        name: name,
+        email: email,
+        contact: phone,
+      },
+    };
+    var paymentObj = new window.Razorpay(options);
+    paymentObj.open();
+    paymentObj.on("payment.failed", function (response) {
+      alert(response.error.code);
```

Comment:
why do you have so many alerts?
alerts and console.logs are to be removed before committing code